### ISSUE TITLE
Is their any way you can make a Version in Java Just for MCPE?

### ISSUE THREAD

**Issue by yungtechboy1** [DiamondCoreProject/DiamondCore#2]
Is their any way you can make a Version in Java Just for MCPE?

**Comment by SuperstarGamer** [comment]
Yes, in fact I have a option planned where you can choose the mode the server runs in.

Global Mode - Allows both MCPE and MCPC clients to connect, restricts features that aren't on both sides
Global MCPE Mode - Does what Global Mode does but only allows MCPE clients to join
Global MCPC Mode - Does waht Global Mode does but only allows MCPC clients to join
MCPE Mode - Only allows MCPE clients to join and has every feature for MCPE
MCPC Mode - Only allows MCPC clients to join And has every feature for MCPC

**Comment by yungtechboy1** [comment]
Cool!
